index.html

```diff
@@ -2188,9 +2188,12 @@
             <code>"NotSupportedError"</code> , and abort these steps.
           </li>
           <li>
-            Otherwise, let <var>watchId</var> be a locally unique number
-            assigned to <var>options</var> and <var>callback</var> and store
-            them together as an <a>NFC watch</a>.
+            Let <var id="watch-suspended">suspended</var> be <code>false</code>.
```

Comment:
This is probably wrong, because it'll override the assignment from "Handling visibility and focus". I think you don't want a `suspended` variable _at all_. Instead, each place it's used, just check whether the window ([responsible browsing context](https://html.spec.whatwg.org/multipage/webappapis.html#responsible-browsing-context)?) is visible and focused.